.circleci/config.yml

```
@@ -299,9 +299,12 @@ commands:
       - save_cache:
           key: pip-v2-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
```

```suggestion
          key: pip-v3-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
```
I think we should bump cache version, since the cache for `master` still includes solc stuff, and we may experience problems when we merge this. In any case, with this change we only will incur in 2 or 3 more minutes for that single build.

---

✔️